server/config/42tester.conf

```diff
@@ -36,12 +40,17 @@ server {
 
 # /post_body must answer anything to POST request with a maxBody of 100
 
-	client_max_body_size 100; MB?
-	client_body_buffer_size 100; MB?
+	client_max_body_size 100; bytes
+	client_body_buffer_size 100; bytes
 
 	location /directory/ {
-		method GET
+		method GET POST
 		root /42tester/YoupiBanane/
 		index_page youpi.bad_extension
 	}
+
+	# location /uploads/ {
```

Comment:
remove comment

Reply:
dc9cc883d2cfd93bc5d7bd35739f1eba3d37851c fixed it